src/leiningen/new/tenzing.clj

```diff
@@ -146,7 +157,9 @@
            (remove nil?
                    (vector (if (divshot? opts) ["divshot.json" (render "divshot.json" data)])
                            (if (garden? opts)  ["src/clj/{{sanitized}}/styles.clj" (render "styles.clj" data)])
-                           (if (sass? opts)    ["sass/styles.sass" (render "styles.sass" data)])
+                           (if (sass? opts)    ["sass/styles.scss" (render "styles.scss" data)])
+
+                           (if (less? opts)    ["less/styles.main.less" (render "styles.main.less" data)])
```

Comment:
The reason for `styles.main.less` is because, as of now, boot-less is specifically looking for a _main_ less file with a prefix of `.main.less`. Perhaps boot-less should except a configuration option for this purpose.